website/sidebars.js

```diff
@@ -1,5 +1,11 @@
+function sorted(a, b) {
```

Comment:
I took the liberty to create this function to order the sidebar alphabetically. It's not changing the order specified in each items categories. Just the root level.

Reply:
This is helpful!